examples/java_postgresql_customvar/README.md

```diff
@@ -178,28 +178,28 @@ make create-backing-db-instance
 
 Now, the only thing that remains is to connect the DB and the application. We let the Service Binding Operator to 'magically' do the connection for us.
 
-Create the following `ServiceBindingRequest`:
+Create the following `ServiceBinding`:
 
 ```shell
 cat <<EOS |kubectl apply -f -
 ---
-apiVersion: apps.openshift.io/v1alpha1
-kind: ServiceBindingRequest
+apiVersion: operators.coreos.com/v1alpha1
+kind: ServiceBinding
 metadata:
   name: binding-request
   namespace: service-binding-demo
 spec:
-  applicationSelector:
-    resourceRef: java-app
+  application:
+    name: java-app
     group: apps
     version: v1
     resource: deployments
-  backingServiceSelector:
-    group: postgresql.baiju.dev
-    version: v1alpha1
-    kind: Database
-    resourceRef: db-demo
-  customEnvVar:
+  - services:
+      group: postgresql.baiju.dev
```

Comment:
A `-` is missing here. And removed from above line

Reply:
https://github.com/redhat-developer/service-binding-operator/pull/495/commits/ae587fc0b9dc139a570cdcfea5e277a7f1e2ba4f thanks!